mono/mini/mini-posix.c

```diff
@@ -1084,12 +1084,21 @@ dump_native_stacktrace (const char *signal, void *ctx)
 		if (pid == 0) {
 			dup2 (STDERR_FILENO, STDOUT_FILENO);
 
+			mono_runtime_printf_err ("\nDebug info from gdb:\n");
 			mono_gdb_render_native_backtraces (crashed_pid);
 			exit (1);
 		}
 
-		mono_runtime_printf_err ("\nDebug info from gdb:\n");
 		waitpid (pid, &status, 0);
+
+#if !defined(DISABLE_CRASH_REPORTING)
+		// We've already done our gdb dump and our telemetry steps. Before exiting,
```

Comment:
The send_crash () function already contains an #if !defined(DISABLE_CRASH_REPORTING)

Reply:
Okay, relying on that one now